pages/blog/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { Layout } from '../../layouts/article/layout';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { load } from 'cheerio';
+import { Blog } from '../../components/blog';
+import { sortBy } from 'lodash';
+
+type BlogType = {
+  url: string;
+  title: string;
+  profilePic: string;
+  date: string;
+  name: string;
+};
+
+type Writer = {
+  name: string;
+  username: string;
+};
+
+const getBlogsByUsernames = async (writers: Writer[]) => {
+  return Promise.all(
+    writers.map(async writer => {
+      const baseUrl = 'https://dev.to';
+      let response;
+      try {
+        response = await axios.get(`${baseUrl}/${writer.username}`);
+      } catch (e) {
+        console.log(e.message);
```

Comment:
wonder if it would be fine to just let this throw if it fuks up, so no try catch at all and just let it error to console. or does it explode the app? :D